clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/VCVTraitSet.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.monster.etl.clinvar.models.output
+
+import io.circe.Encoder
+import io.circe.derivation.{deriveEncoder, renaming}
+import org.broadinstitute.monster.etl.clinvar.models.intermediate.WithContent
+import upack.Msg
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+  * Info about a collection of traits in ClinVar.
+  *
+  * @param id unique ID of the trait collection
+  * @param `type` common type of the trait collection
+  */
+case class VCVTraitSet(id: String, `type`: Option[String], traitIds: ArrayBuffer[String])
```

Comment:
Oh, I see what you mean now. This field should be an `Array`:
```suggestion
case class VCVTraitSet(id: String, `type`: Option[String], traitIds: Array[String])
```
And then the code below can accept an `Array`